src/components/common/Extensions/UserHub/UserHub.tsx

```diff
@@ -111,7 +111,7 @@ const UserHub: FC<UserHubProps> = ({
           </>
         )}
       </div>
-      <div className={clsx('relative h-full w-full')}>
+      <div className="relative h-full w-full min-w-0">
```

Comment:
`min-w-0` and `max-w-full` classes added for different elements to make `text-overflow: ellipsis` on pills actually works

<img width="460" alt="Screenshot 2024-07-17 at 15 54 35" src="https://github.com/user-attachments/assets/36ef7e49-44dc-4614-87bb-7e26e2f5d85e">